### PR TITLE
Added the Test run time to the Locust UI

### PR DESCRIPTION
**Pull request by SecretObsession**
This PR will add the run time to the Locust Web UI.

### CODE REVIEW

**Comment by DanielWebster** [comment]
+1

File: locust/stats.py [hunk 1]
```diff
@@ -63,7 +64,13 @@ def clear_all(self):
...
         self.last_request_timestamp = None
         self.start_time = None
-        
+
+    def total_run_time(self):
+        try:
+            # return the time that the test has been running for.
+            self.run_time = int(time.time() - self.start_time)
+        except Exception as ex:
+            print(ex)
```
Comment (cgoldberg) [inline locust/stats.py]: is it useful to just print the exception and continue like this?
this exception doesn't seem like it could ever be raised here.

**Comment by SecretObsession** [comment]
@cgoldberg Thanks, you're right, it's been removed.

**Comment by mbeacom** [comment]
@SecretObsession Thank you for the contribution! Would you mind rebasing this branch?

**Comment by cyberw** [comment]
Hi @SecretObsession ! This PR has been idle for very long and has conflicts, so I am closing it. Feel free to reopen if this is still important to you!

**Pull request closed without merging**